ui/lib/kv/addon/routes/secret.js

```diff
@@ -39,6 +39,7 @@ export default class KvSecretRoute extends Route {
       secret: this.fetchSecretData(backend, path),
       subkeys: this.fetchSubkeys(backend, path),
       metadata: this.fetchSecretMetadata(backend, path),
+      canPatchSecret: this.capabilities.canPatch(`${backend}/data/${path}`),
```

Comment:
I can do this in the component instead if folks prefer. This just made component testing easier.

Reply:
I think following the standard we've leaned into here makes more sense than starting a new one in this instance